src/Elastic.Markdown/Slices/Layout/_Head.cshtml

```diff
@@ -4,8 +4,9 @@
 	<link rel="stylesheet" type="text/css" href="@Model.Static("styles.css")"/>
 	<meta charset="utf-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	@await RenderPartialAsync(_Favicon.Create())
 	<meta name="robots" content="@(Model.AllowIndexing ? "index, follow" : "noindex, nofollow")">
-	<meta name="htmx-config" content='{"scrollIntoViewOnBoost": false, "selfRequestsOnly": true}'>
```

Comment:
Unrelated change: We are not using boosted links. Hence, this config option is not necessary.